app/src/main/java/org/schabi/newpipe/fragments/detail/VideoDetailFragment.java

```diff
@@ -672,8 +676,8 @@ protected void initListeners() {
             if (motionEvent.getAction() == MotionEvent.ACTION_DOWN) {
                 animate(binding.touchAppendDetail, true, 250, AnimationType.ALPHA,
                         0, () ->
-                        animate(binding.touchAppendDetail, false, 1500,
-                                AnimationType.ALPHA, 1000));
+                                animate(binding.touchAppendDetail, false, 1500,
+                                        AnimationType.ALPHA, 1000));
```

Comment:
And this

Reply:
I am not able to resolve this , if you can request the changes I'll merge it and will be thankful if you help me out

Reply:
If i don't reformat the code , it gives checkstyle violation :(